### PR TITLE
IGNITE-10810: Import models from MLeap

### PR DESCRIPTION
**Pull request by dmitrievanthony**

### CODE REVIEW

File: modules/ml/mleap-model-parser/src/main/java/org/apache/ignite/ml/mleap/MLeapModel.java [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+    }
+
+    /**
+     * Makes a prediction using default column order specified in {@link #schema}.
+     *
+     * @param input Input arguments.
+     * @return Prediction result.
+     */
+    public double predict(Double[] input) {
+        LeapFrameBuilder builder = new LeapFrameBuilder();
```
Comment (avplatonov) [inline modules/ml/mleap-model-parser/src/main/java/org/apache/ignite/ml/mleap/MLeapModel.java]: add assertion that input.size == schema.size
and just use such behavior:
HashMap vec = IntStream.range(0, input.length).boxed().collect(Collectors.asMap(i -> schema.get(i), i -> input.get(i)));
return predict(vec);

File: modules/ml/mleap-model-parser/src/main/java/org/apache/ignite/ml/mleap/MLeapModel.java [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+    /**
+     * Makes a prediction using MLeap API.
+     *
+     * @param inputFrame Input MLeap frame.
+     * @return Prediction result.
+     */
+    public double predict(DefaultLeapFrame inputFrame) {
+        DefaultLeapFrame outputFrame = transformer.transform(inputFrame).get();
+
+        Try<DefaultLeapFrame> resFrame = outputFrame.select(new Set.Set1<>("price_prediction").toSeq());
```
Comment (avplatonov) [inline modules/ml/mleap-model-parser/src/main/java/org/apache/ignite/ml/mleap/MLeapModel.java]: was ist das???? ("price_predicion")

File: examples/src/main/java/org/apache/ignite/examples/ml/mleap/MLeapModelParserExample.java [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+        try (Ignite ignite = Ignition.start("examples/config/example-ignite.xml")) {
+            File mdlRsrc = IgniteUtils.resolveIgnitePath(TEST_MODEL_RES);
+            if (mdlRsrc == null)
+                throw new IllegalArgumentException("File not found [resource_path=" + TEST_MODEL_RES + "]");
+
+            ModelReader reader = new FileSystemModelReader(mdlRsrc.getPath());
+
+            AsyncModelBuilder mdlBuilder = new IgniteDistributedModelBuilder(ignite, 4, 4);
+
+            try (Model<HashMap<String, Double>, Future<Double>> mdl = mdlBuilder.build(reader, parser)) {
```
[the remark this answers is not in the document]
Comment (dmitrievanthony) [reply examples/src/main/java/org/apache/ignite/examples/ml/mleap/MLeapModelParserExample.java]: Added correspondent TODO and JIRA: https://issues.apache.org/jira/browse/IGNITE-10834.

File: modules/ml/mleap-model-parser/src/main/java/org/apache/ignite/ml/mleap/MLeapModel.java (hunk 1 above)
Comment (dmitrievanthony) [reply modules/ml/mleap-model-parser/src/main/java/org/apache/ignite/ml/mleap/MLeapModel.java]: Done.

File: modules/ml/mleap-model-parser/src/main/java/org/apache/ignite/ml/mleap/MLeapModel.java (hunk 2 above)
Comment (dmitrievanthony) [reply modules/ml/mleap-model-parser/src/main/java/org/apache/ignite/ml/mleap/MLeapModel.java]: Guter punkt, korrigiert.

File: modules/ml/mleap-model-parser/src/main/java/org/apache/ignite/ml/mleap/MLeapModel.java [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+     *
+     * @param input Input arguments.
+     * @return Prediction result.
+     */
+    public double predict(Double[] input) {
+        if (input.length != schema.size())
+            throw new IllegalArgumentException("Input size is not equal to schema size");
+
+        Map<String, Double> vec = IntStream.range(0, input.length)
+            .boxed()
```
Comment (avplatonov) [inline modules/ml/mleap-model-parser/src/main/java/org/apache/ignite/ml/mleap/MLeapModel.java]: maybe it "boxed()" should be on same line  as IntStream
it seems to be empty

**Pull request closed without merging**